src/Libraries/Microsoft.Extensions.Compliance.Abstractions/README.md

```diff
@@ -1 +1,65 @@
-README
+# Microsoft.Extensions.Compliance.Abstractions
+
+This provides types necessary to extend the redaction feature.
+
+## Install the package
+
+From the command-line:
+
+```dotnetcli
+dotnet add package Microsoft.Extensions.Compliance.Abstractions
+```
+
+Or directly in the C# project file:
+
+```xml
+<ItemGroup>
+  <PackageReference Include="Microsoft.Extensions.Compliance.Abstractions" Version="[CURRENTVERSION]" />
+</ItemGroup>
+```
+
+## Usage Example
+
+### Implementing Redactors
```

Comment:
In general, I don't expect developers to implement their own redactors, so this would definitely be a niche use case...